### PR TITLE
pricing + github-student-developer-pack updates

### LINKED ISSUE #481: [pricing] rearranging pricing cards
>Currently, it's very unclear to many users that:
>Gitpod can be used for free, right now
>Unlimited Open Source is not an Unlimited plan

>Suggestions:
>Make our free Open Source plan more obvious by re-adding its plan card
>Make the 1 month private repo trial more obvious
>Rename the Unlimited Open Source plan to avoid false expectations

mockup:
Bildschirmfoto 2020-02-12 um 17 01 08

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #481

### CODE REVIEW

**Comment by nisarhassan12** [comment]
> - [ ] break points need to be adjusted: we could use a vertical scroll instead of narrowing the cards.-  Could you implement this?
image

I can do this and this would work for large mobile devices, but we have to again display them as rows in a single column on small devices because there are 5 tabs (The above mockup is missing the 5th tab for Enterprise) .We could make the tab panels font-size small but that would result in small touch/tap areas and this can be problematic for accessibility:

image

Also, would it be ok if I were to have a scroll bar instead of the tabbed panels like we have in other places?

File: src/pages/pricing.tsx [hunk 1]
```diff
@@ -37,13 +47,17 @@ const plans: PricingBoxProps[] = [
         duration: 'unlimited hours',
         features: ['Private & Public Repos', <span className="span">8 Parallel Workspaces <PopOver description="The number of workspaces running at the same time."/></span>, <span className="span">Team Manageable&nbsp;<PopOver description="Setup Gitpod for an entire team with a single invoice and credit card."/></span>, <span className="span">30min Timeout <PopOver description="Workspaces without user activity are stopped after 30 minutes."/></span>],
         transform: 'scale(1.05)',
+        banner: 'Recomended',
```
Comment (svenefftinge) [inline src/pages/pricing.tsx]: recommended

**Comment by svenefftinge** [comment]
I would prefer to have the background of the open-source plan white as well. It looks a but alien/secondary otherwise.

**Comment by ChristinFrohne** [comment]
>Also, would it be ok if I were to have a scroll bar instead of the tabbed panels like we have in other places?

The thought was to have the horizontal scrolling on tablets, but I saw that you adjusted the width of the boxes and made them bigger on tablet size. That's great. If the horizontal scrolling in rather difficult to implement, we can also keep it like this. 

Great that you changed the overlay of the tooltips. It's lots better this way :) 

- [ ] Please list the open-source plan first and use the same spacing for the boxes below

Capture

- [ ] can you have the section with the features/ticks arranged in the center of each box? The text should stay left aligned

- [ ] could you use an earlier break point for this view?
Capture2

**Comment by nisarhassan12** [comment]
I have made the changes this is how they look now:

image

Macking the space equal between the bottom ones is not that easy. Do they look good now if not then I can give another try :slightly_smiling_face: .

**Pull request merged**